core/src/main/java/com/sequenceiq/cloudbreak/controller/v4/ClusterTemplateV4Controller.java

```diff
@@ -139,9 +148,8 @@ public ClusterTemplateV4Responses deleteMultiple(Long workspaceId, @ResourceName
         if (Objects.nonNull(names) && !names.isEmpty()) {
             clusterTemplates = clusterTemplateService.deleteMultiple(names, workspaceId);
         } else {
-            NameOrCrn environmentNameOrCrn = NameOrCrn.ofCrn(environmentCrn);
             Set<String> namesByEnv = clusterTemplateService
-                    .findAllByEnvironment(environmentNameOrCrn)
+                    .findAllByEnvironment(environmentCrn)
```

Comment:
if you remove the input-independent solution (the usage of the NameOrCrn object) then please remove the unused environmentName arg also from the method arguments

Reply:
Yes, it could be valid, but:
* only one place where this is called: `ClusterTemplateV4Controller.deleteMultiple -> ClusterTemplateService.findAllByEnvironment -> and call this function`
* if you see the deleteMultiple function, we never use the `name`:
```
NameOrCrn environmentNameOrCrn = NameOrCrn.ofCrn(environmentCrn);
Set<String> namesByEnv = clusterTemplateService
        .findAllByEnvironment(environmentNameOrCrn)
        .stream()
        .map(ClusterTemplate::getName)
        .collect(toSet());
```
* and this chain references causes this:
   `env service call multiple delete with crn -> we fetch the detailed env with this crn -> and use the name and crn to log` , does not look good.

I think nobody call with name because of last change about there January 2020.

Reply:
I have nothing against removing the usage of NameOrCrn from this method, my comment just pointed to the fact that we shouldn't let unused method arguments as we do in this case after this change